### ISSUE TITLE
Greek letter displays break up command wrong

### ISSUE THREAD

**Issue by jvdwetering** [tikzit/tikzit#76]
When you have a node with the text `\pi`, it is replaced by a unicode π. However, if I were to have a custom command called, say, `\pic`, then this would display as πc, which is obviously not was intended.

**Comment by akissinger** [comment]
Yeah, I knew about this one. I've been weighing this minor annoyance vs. the slightly larger annoyance of writing a latex tokenizer in C++.

**Comment by jvdwetering** [comment]
Could an intermediate solution be to detect whether the next character is alphabetical?
So if you match on \pi, check if the immediate next character is in a-zA-Z, and then abort. My guess is this should remove most of the issues
